### ISSUE TITLE
[question] Can I customize policy network with keras of tensorflow.layers?

### ISSUE THREAD

**Issue by zkx741481546** [hill-a/stable-baselines#817]
**Describe the bug**
I'm hoping to use my own network structure rather than those Policy in this framework, so that I can use BN, Dropout, and  Regularization easily.
Or can I set these things with custom Policy, and how?

**Comment by Miffyli** [comment]
Take a look at the [documentation](https://stable-baselines.readthedocs.io/en/master/), specifically on [custom policies](https://stable-baselines.readthedocs.io/en/master/guide/custom_policy.html).
